### PR TITLE
76 add mail server

### PR DESCRIPTION
**Pull request by JohanSaisa**

### CODE REVIEW

**Comment by JohanSaisa** [comment]
[US2003: Användare - Inlogg via sociala medier](https://trello.com/c/Ti79gVWs/20-us2003-anv%C3%A4ndare-inlogg-via-sociala-medier)

**Review by RasmusBroborg — CHANGES_REQUESTED** [review]

File: src/GT.Core/Services/Impl/EmailSender.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	public class EmailSender : IEmailSender
+	{
+		private readonly IConfiguration _configuration;
+
+		public EmailSender(IConfiguration configuration)
+		{
+			_configuration = configuration;
+		}
+
+		public async Task SendEmailAsync(string email, string subject, string htmlMessage)
```
[the remark this answers is not in the document]
Comment (JohanSaisa) [reply src/GT.Core/Services/Impl/EmailSender.cs:25]: Default naming convention from implemented interface IEmailSender - propose we keep this aligned.

File: src/GT.Core/Services/Impl/AuthMessageSenderOptions.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace GT.Core.Services.Impl
+{
+	public class AuthMessageSenderOptions
+	{
+		public string? SendGridKey { get; set; }
```
[the remark this answers is not in the document]
Comment (JohanSaisa) [reply src/GT.Core/Services/Impl/AuthMessageSenderOptions.cs:5]: Removed in new commit. Residual from when looking to implement Send Grid according to best practice - settled for SMTP as of now.

**Review by antoncarlgren — APPROVED** [review]

**Review by RasmusBroborg — APPROVED** [review]

File: src/GT.UI/Program.cs [hunk 3]
```diff
@@ -59,6 +60,10 @@
 	.AddTransient<IGTListingInquiryService, GTListingInquiryService>()
 	.AddTransient<IGTExperienceLevelService, GTExperienceLevelService>();
 
+//Add Email service
+builder.Services
+	.AddTransient<IEmailSender, EmailSender>();
```
[the remark this answers is not in the document]
Comment (JohanSaisa) [reply src/GT.UI/Program.cs]: Fixed.

**Pull request merged**